src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Square extends Figure implements AreaCalculetion {
+    private double side;
+
+    public Square(double side, String color, String name) {
+        this.side = side;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    @Override
+    public double getArea() {
+        return getSide() * 2;

```

Comment:
The formula for the area calculation is incorrect. For a square, the area is the side length squared ('side * side' or 'Math.pow(side, 2)').